### PR TITLE
Properties Editor Structure and Style

### PR DESCRIPTION
**Pull request by Gazook89**
This is mostly a small refactoring of CSS and some slight changes to HTML structure in advance of the new TagInput component.  Things I noticed while working on that PR are the many places that font sizes were set, and the variety of font-size values used in different places.  And this also updates the "publish" button description, some box sizing and borders, etc.

No methods/functions were harmed in the production of this PR.  

I'd like to get this pushed through before doing much styling on the TagInput component update.

### CODE REVIEW

File: client/homebrew/editor/metadataEditor/metadataEditor.jsx [hunk 1]
```diff
@@ -304,17 +304,14 @@ const MetadataEditor = createClass({
...
-				</a>
+				<small><a href='/legacy' target='_blank' rel='noopener noreferrer'>Click here to see the demo page for the old Legacy renderer!</a></small>
 			</div>
 		</div>;
 	},
 
 	render : function(){
 		return <div className='metadataEditor'>
-			<h1 className='sectionHead'>Brew</h1>
+			<h1 className='sectionHead'>Properties Editor</h1>
```
Comment (calculuschild) [inline client/homebrew/editor/metadataEditor/metadataEditor.jsx:314]: This is the title of the section, not the whole tab (i.e along with "authors" and "privacy")

**Comment by calculuschild** [comment]
Is this ready for merging or do you want to tinker on it any more?

File: client/homebrew/editor/metadataEditor/metadataEditor.jsx (hunk 1 above)
Comment (Gazook89) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:314]: I believe it’s fine the way I have it. Many things in life have an H1, then some content, then are followed with h2 and more content. 

Nothing is really lost by removing “brew” from the top here. Everything in that panel is about the brew. We don’t alter anything else with the units like the account. 

But, we do frequently reference the Properties Editor when helping users, and it’d be nice to have it labeled (beyond the hover state on the bar). 

Just my two cents

Comment (calculuschild) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:314]: If you want a top-level title that says "Properties Editor", that is fine. But I meant that having this element on the same level as "Authors" and "Privacy" implies that only part of this panel is the Properties Editor, when "Properties Editor" should apply to the whole page. We are missing one higher-level header where "Properties Editor" belongs.

An equivalent menu at a restaurant:

### Sandwiches (shouldn't this be "Breads", and "Sandwiches" at a higher level at the top?)

- Whole Wheat
- Pumpernickle
- White

### Vegatables

- Lettuce
- Tomato

### Meats

- Turkey
- Bacon

Comment (Gazook89) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:314]: Okay, so just leave Properties Editor as H1, and demote "Privacy" etc to H2

Comment (Gazook89) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:314]: What about this:

image

Comment (calculuschild) [reply client/homebrew/editor/metadataEditor/metadataEditor.jsx:314]: That's fine, yes. Thanks!

**Pull request merged**